app/styles/modules/_settings.scss

```diff
@@ -573,3 +573,49 @@ section.modal-panel {
     margin-bottom: 30px;
   }
 }
+
+.fxa-radio {
```

Comment:
And the rest of the styles `ltr / rtl + label styles` can be under `fxa-radio-group` class